docusaurus/docs-internals/editor-controls/data-controls/multiselect.md

```diff
@@ -0,0 +1,238 @@
+---
+toc_max_heading_level: 4
+---
+# Multi Select
+
+The multiselect control allows users to select multiple options from a list, displaying the number of selected items and the individual selections. It offers both synchronous and asynchronous loading of choices, enhancing flexibility and user experience.
+
+Example of the multiselect without selection:
+
+![Multiselect Deselected](/img/data-controls/multiselect-deselected.png)
+
+Example of the multiselect with single selection:
+
+![Multiselect Selected1](/img/data-controls/multiselect-selected1.png)
+
+Example of the multiselect with multiple selections:
+
+![Multiselect Selected2](/img/data-controls/multiselect-selected2.png)
+
+### Parameters
```

Comment:
missing keys in parameters: 
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/6def1936-8bb7-491b-ada5-ec89c3c4354a)